examples/plot_function_counter.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+"""
+======================================
+Identifying function names in a script
+======================================
+
+Calls Sphinx-Gallery identify names function to figure out which
+functions are called in the script and to which module do they belong.
+"""
+
+# Code source: Óscar Nájera
+# License: BSD 3 clause
+
+import matplotlib.pyplot as plt
+import sphinxgallery.backreferences as spback
+
+
+filename = spback.__file__.replace('.pyc', '.py')
+file_code = open(filename, 'r').read()
+names = spback.identify_names(file_code)
+figheight = (len(names)+.5)
+
+fontsize = 0.3 * 72
```

Comment:
Because that's how it is shown in the original example I copied from in matplotlib. Getting the fonts big enough for the figure. But putting 22 is better.